src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -69,31 +69,36 @@ type Props = ProvidedProps & StoreProps;
 
 function mapStateToProps() {
   /** Gets items for the loadout builder and creates a mapping of classType -> bucketHash -> item array. */
-  const itemsSelector = createSelector(allItemsSelector, (allItems): Readonly<{
```

Comment:
I've been getting some weird formatting changing coming from the git hook lately. It seems my vscode formatting clashes with it.

Reply:
Looking at it, I think pretty-quick is not respecting the `.prettierrc` file...maybe. If i delete the line width config in `.prettierrc` (so it uses default 80), I get this same change happening. This line is 80 chars plus a character return.

Reply:
Actually it might be an update to vs code and thus the prettier extensions. Now this formatting is respected by vs code on save since the restart and update.